docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,55 @@
 # Development Environment
 
-The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+This guide is for setting up your local environment for JavaScript development for creating plugins and tools to extend WordPress and the block editor.
 
-A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
+A development environment is a catch-all term for what you need setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
-## Development Tools
+## Quickstart
+
+Here is a summary of the guide. See each section for additional details and explanations.
+
+**1. Install Node development tools**
+
+Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Quit and restart terminal
+```
+nvm install --lts
+```
+
+**2. WordPress Development Site**
+
+First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) following the instructions for your OS.
+
+- Install WordPress environment tool
+
+```
+npm -g install @wordpress/env
+```
+
+Start the environment:
+
+ ```
+ wp-env start
+ ```
+
+You will have a full WordPress site installed, navigate to: http://localhost:8888/ the defualt user is "admin" and default password is "password"
+
+
+**3. Code Editor**
+
+You can use any code editor, [Visual Studio Code](https://code.visualstudio.com/) is a popular open-source tool, follow instructions on their site for your OS.
+
```

Comment:
See also the Code Editor section below for additional information. (As we here need to let the user know that there is another Code Editor section below. I am wondering if these should be merged into one Code Editor section. It feels a bit strange having two.)